os-downstream-tag.py

```diff
@@ -111,14 +111,14 @@ def main():
     fetch(downstream_remote)
 
     upstream_refs = [
-        "stable/{}".format(release),
-        "unmaintained/{}".format(release),
+        "{}/stable/{}".format(upstream_remote, release),
+        "{}/unmaintained/{}".format(upstream_remote, release),
         "{}-eol".format(release),
```

Comment:
Does the eol ref not need the upstream remote as well?

Reply:
That's the point of the change. See the explanation in the commit message.

Reply:
:man_facepalming: Yep, I get it now